### ISSUE TITLE
Image description is not being written

### ISSUE THREAD

**Issue by mutil** [Zazama/node-id3#82]
Hi and thank you for this nice library.
When trying to write tags to a file, image description seems to remain undefined.

Sample code:
```
let tags = {
  title: 'Title',
  image: {
    description: 'test',
    type: { id: 3, name: 'front cover' },
    mime: 'jpg',
    imageBuffer: Buffer.from(base64, 'base64'),
  }
} 

NodeID3.create(tags);
NodeID3.write(tags, fileName);

const data = NodeID3.read(fileName);
console.log(data);
```

**Comment by Zazama** [comment]
Hey,
thanks for the report, you're right, description actually isn't written at all. I will push a fix soon.

**Comment by Zazama** [comment]
Fixed in 0.1.20
